episodes/server_resources.md

```diff
@@ -0,0 +1,211 @@
+
+:::{dropdown} Solution
+*Small: 4 CPU, 32 GB RAM* – the task failed on a *Shared Small* machine and we recommend sizing up incrementally if you are unsure of the server resources you need.
+
+:::
+
+::::
+
+### Automatic server culling
+
+Hubs automatically cull servers after a period of inactivity in order to improve resource utlization and safeguard against excessive cloud costs. This occurs after **1 hour** of user inactivity.
+
+:::{tip}
+Executing a piece of long-running code with a running kernel is not recognised as user activity on the server and so the process is killed after 1 hour. This is a rare use case of interactive computing on a hub, but this limit can be reconfigured for a particular hub upon request by contacting [2i2c support](#). 
+
+<!-- TODO: Add internal reference for contacting 2i2c support when written up. -->
+:::
+
+
+(episode-server:shutdown-server)=
+## Shutting down a server as a user
+
+A user should shut down their own server on the hub once they are finished with their work. This can be done manually by applying the following steps.
+
+1. (JupyterLab) Navigate to *File > Hub Control Panel*, or alternatively (JupyterLab and RStudio) navigate to the URL `https://<your-hub-url>.2i2c.cloud/hub/home`
+1. In the Hub Control Panel, click on the ![Stop My Server](../media/episodes/server_resources/shutdown-2.png) button
+
+:::{image} ../media/episodes/server_resources/shutdown-1.png
+:width: 100%
+:alt: Screenshot showing the File > Hub Control Panel menu option in the JupyterLab user interface.
+
+:::
+
+## Managing another user's server as a Hub Champion
+
+Hub Champions have access to an Admin panel from the Hub Control Panel (or the URL `https://<your-hub-url>.2i2c.cloud/hub/admin`).
+
+:::{image} ../media/episodes/server_resources/manage-1.png
+:width: 100%
+:alt: Screenshot showing the Admin panel available to Hub Champions.
+
+:::
+
+::::{caution}
+:::{epigraph}
+With great power comes great responsibility.
+
+-- Spiderman
+:::
+Performing administrative actions as a Hub Champion should be done with great care. Respect the privacy of others and think before you act.
+
+::::
+
+### Stop and restart a user's server
+
+A user's server may sometimes become unresponsive, e.g. issues stemming from self-installed software via `pip` and/or `conda`. Hub Champions can restore user servers to their default environment by stopping and restarting their server by clicking the ![Stop Server](../media/episodes/server_resources/manage-2.png) followed by the ![Start Server](../media/episodes/server_resources/manage-3.png) button next  to the user's name from the Admin panel.
+
+:::{note}
+Self-installed software packages on the hub are *temporary* and are removed each time a server is shut down to maintain a clean default software environment. Software is persisted between sessions by {ref}`episode-images`, which is covered in the next episode.
```

Comment:
"Persisted" feels awkward here. Is this a common usage of the verb that I am unfamiliar with?

Reply:
That's fair – I can change this to

```suggestion
Self-installed software packages on the hub are *temporary* and are removed each time a server is shut down to maintain a clean default software environment. Software is provided between sessions by {ref}`episode-images`, which is covered in the next episode.
```